benchmarks/imagenet/vitb16/finetune_eval.py

```diff
@@ -127,6 +128,7 @@ def configure_optimizers(self):
 
 def finetune_eval(
     model: Module,
+    method: str,
```

Comment:
Should we add a `finetune_method` argument instead of `method`? Then users can set `--finetune-method=mae`  if they want to run the MAE fine-tuning recipe independent of the pretraining method.